api/r/CellCensus/CellCensus.Rproj

```diff
@@ -0,0 +1,22 @@
+Version: 1.0
```

Comment:
I don't pretend to know R ecosystem versioning conventions, but should an early release such as this start as a 1.0?  Or 0.?

Reply:
Also should we sync this to the Python versioning? We should probably consolidate the behavior with tiledb-soma anyway.

Reply:
This `1.0` is a red herring -- it describes the *format/schema version* of the `.Rproj` file, not of the project/package itself. The package version is written in the `DESCRIPTION` file and defaulted to `0.0.0.9000` (I don't know where the 9000 comes from!). Agree we will want to some some coordinated version numbers, hopefully controlled by git tags, when we're settled enough to make coherent release versions. Looking forward to that =)